### PR TITLE
Added links to result page for Slack #1519

### PR DESCRIPTION
**Pull request by soulgalore**

### CODE REVIEW

File: lib/plugins/slack/attachements.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function getMetric(metric) {
   }
 }
 
-module.exports = function(dataCollection, slackOptions) {
+module.exports = function(dataCollection, slackOptions, resultBaseURL, storageManager) {
```
Comment (tobli) [inline lib/plugins/slack/attachements.js]: I think it'd be good to place the slackOptions map last (that's the pattern we use for other functions that take options).

File: lib/plugins/slack/summary.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const util = require('util');
 const get = require('lodash.get');
 const h = require('../../support/helpers');
 
-module.exports = function(dataCollection, errors, options) {
+module.exports = function(dataCollection, errors, options, storageManager) {
```
Comment (tobli) [inline lib/plugins/slack/summary.js]: options last, as above.

File: lib/plugins/slack/attachements.js [hunk 3]
```diff
@@ -108,9 +108,16 @@ module.exports = function(dataCollection, slackOptions) {
...
+    if (resultBaseURL) {
+      const baseDir = storageManager.getRelativeBaseDir();
+      const pagePath = storageManager.pathFromRootToPageDir(url);
+      message = ` (<${resultBaseURL}/${baseDir}/${pagePath}|result>)`;
+    }
+
     attachments.push({
       color,
-      text: url,
+      text: url + message,
```
Comment (tobli) [inline lib/plugins/slack/attachements.js]: Maybe if the message variable was called text, initiated to url and appended to inside the if-statement. Then the attachment object could consistently use shorthand syntax.

File: lib/plugins/slack/attachements.js [hunk 4]
```diff
@@ -108,9 +108,16 @@ module.exports = function(dataCollection, slackOptions) {
       }
     }
 
+    let message = '';
+    if (resultBaseURL) {
+      const baseDir = storageManager.getRelativeBaseDir();
```
Comment (tobli) [inline lib/plugins/slack/attachements.js]: I think we should reconsider the approach with relativeBaseDir. It's not a good fit for use with outputFolder. ```--resultBaseURL="http://example.com/bar" --outputFolder="../../foo"``` will result in ```http://example.com/foo/...```when formatted by Slack. ```--resultBaseURL="http://example.com/bar" --outputFolder="/var/www/foo"``` becomes ```http://example.com/bar//var/www/foo``` which is probably not what the user wanted.

Comment (soulgalore) [inline lib/plugins/slack/attachements.js]: We have https://github.com/sitespeedio/sitespeed.io/blob/720d3b93c21034ce4ded3707916276a92888fb1a/lib/plugins/s3/index.js#L39 that makes the combo of S3 and outputFolder a combo that doesn't work. I think we should create another issue to fix this longterm

Comment (tobli) [inline lib/plugins/slack/attachements.js]: Ahh, I see. Sounds like a plan. Merge this, create a follow up issue.

Comment (soulgalore) [inline lib/plugins/slack/attachements.js]: https://github.com/sitespeedio/sitespeed.io/issues/1527 let us discuss it there!

**Pull request merged**